lib/ProductOpener/DataQualityFood.pm

```diff
@@ -1394,6 +1394,21 @@ sub check_nutrition_data ($product_ref) {
 	return;
 }
 
+	# Check for Mozzarella category and minimum number of ingredients
+	sub check_mozzarella_ingredients {
+		my ($product_ref) = @_;
+
+		if (defined $product_ref->{category_id2} && $product_ref->{category_id2} eq 'mozzarella') {
```

Comment:
We don't want to hardcode mozzarella, instead we should add a property for the corresponding category in the taxonomies/food/categories.txt taxonomy

You can see how it's done above for other checks:

		# some categories have an expected ingredient - push data quality error if ingredient differs from expected ingredient
		# note: we currently support only 1 expected ingredient
		my ($expected_ingredients, $category_id2)
			= get_inherited_property_from_categories_tags($product_ref, "expected_ingredients:en");

Reply:
Alright thanks, I will update my implementation